aiohttp_jinja2/__init__.py

```diff
@@ -18,8 +18,8 @@
 
 
 def setup(app, *args, app_key=APP_KEY, context_processors=(),
-          filters=None, default_helpers=True, **kwargs):
-    env = jinja2.Environment(*args, **kwargs)
+          filters=None, default_helpers=True, enable_async=True, **kwargs):
+    env = jinja2.Environment(*args, enable_async=enable_async, **kwargs)
```

Comment:
should I make this keyword only argument?

Reply:
`enable_async` disabled by default?

Reply:
surely we should check and find out how big the performance impact is? If it's negligible then it can be enabled by default, else not.

Reply:
I'll try running aio-libs/aiohttp-benchmarks with and without async on the fortunes test and see what happens.